plugins/octant/cmd/antrea-octant-plugin/traceflow.go

```diff
@@ -702,10 +713,21 @@ func (p *antreaOctantPlugin) traceflowHandler(request service.Request) (componen
 		Title: "Generate Trace Graph",
 		Form:  graphForm,
 	}
+
+	// Run the previous traceflow again.
+	tAgainForm := component.Form{Fields: []component.FormField{
+		component.NewFormFieldHidden("action", runTraceAgain),
+	}}
+	runTagain := component.Action{
```

Comment:
The variable should be `runTAgain` to meet the [mixedCaps](https://golang.org/doc/effective_go#mixed-caps) converntion. But maybe just use a full name here is more clear - `runTraceAgain`.